src/Microsoft.OData.Core/UriParser/SemanticAst/ODataPath.cs

```diff
@@ -90,10 +90,7 @@ internal IList<ODataPathSegment> Segments
             get { return this.segments; }
         }
 
-        /// <summary>
-        /// Get the segments enumerator
-        /// </summary>
-        /// <returns>The segments enumerator</returns>
+

```

Comment:
Since `GetEnumerator` is a public method, you shouldn't had deleted the `<summary>` and `<returns>` tags